### PR TITLE
api feedback

### PR DESCRIPTION
**Pull request by sandy081**
- remove critical log level
- move log level off to be 0

### CODE REVIEW

**Review by jrieken — DISMISSED** [review]

File: src/vs/platform/log/node/spdlogLog.ts [hunk 1]
```diff
@@ -7,6 +7,16 @@ import * as spdlog from 'spdlog';
 import { ByteSize } from 'vs/platform/files/common/files';
 import { AbstractMessageLogger, ILogger, LogLevel } from 'vs/platform/log/common/log';
 
+enum SpdLogLevel {
+	Trace,
+	Debug,
+	Info,
+	Warning,
+	Error,
+	Critical,
```
Comment (rzhao271) [inline src/vs/platform/log/node/spdlogLog.ts:16]: Is the `Critical` value here ever used?

**Comment by rzhao271** [comment]
The following tests having to do with tasks fail:

https://dev.azure.com/vscode/VSCode/_build/results?buildId=77200&view=logs&j=a8a6689c-5a84-59c0-482a-7dd9f21eb2d8&t=e0b91bb7-6ac8-53ba-6cba-4d995541fd36&l=1174

I'm not sure if the tests are just flaky, but other Code-OSS runs are green, so there might be a regression from this change. Re-running the CI once more to confirm, anyway.

**Comment by S1983tt** [comment]
65589f52422cfb8105c6071c3b439d2510f0e5a0

**Comment by S1983tt** [comment]
b4f57fd8a21ad7b733d4ba1d116f0b8217ff0611

**Comment by sandy081** [comment]
> the Code-OSS CI is still failing. Moving this PR to the next milestone.

I wonder why these test are failing. Changes are not at all related. It is just removing the support for critical logging.

@meganrogge Any insights why tasks tests are failing here?

File: src/vs/platform/log/node/spdlogLog.ts [hunk 2]
```diff
@@ -7,6 +7,16 @@ import * as spdlog from 'spdlog';
...
+	Off
```
Comment (rzhao271) [inline src/vs/platform/log/node/spdlogLog.ts:17]: Test comment

**Review by lramos15 — APPROVED** [review]

**Pull request merged**